packages/psss/src/components/EditableTable.js

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
This is an update to the EditableTable from ui components to use react hook form. It would be good to port this change back to ui components so I've added a not to an upcoming related ticket to do so.

https://app.zenhub.com/workspaces/sprint-board-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/1412